mautic/grapesjs-preset-mautic
# Update grapesjs compatibility (Mautic 4)

Registering your custom components using the legacy method breaks when updating GrapesJS to [release 0.20.1](https://github.com/GrapesJS/grapesjs/releases/tag/v0.20.1). Need this to be able to upgrade GrapesJS in the GrapesJSBuilderBundle.

[there is a PR for M5](https://github.com/mautic/grapesjs-preset-mautic/pull/49)
[there's a PR for M5 with the recent CKE5 changes, this has prio](https://github.com/mautic/grapesjs-preset-mautic/pull/51)

To test:
* have a Mautic 4.x running locally
* replace the package with my forked branch in plugins/GrapesJsBuilderBundle, eg. by running `npm i grapesjs-preset- mautic@github:LordRembo/grapesjs-preset-mautic#update-grapesjs-m4` inside that bundle
* rebuilding GrapesJs bundle by running `npm run build`
* Make or edit an email and see that you can still use drag & drop to add/move/edit a Dynamic content Block.
* You should still be able to save that edited mail
